build_templates/mkmf.template.quikscat.intel

```diff
@@ -0,0 +1,38 @@
+# Template for quikscat converter with Intel Fortran Compiler on Linux or OSX
+#
+# DART software - Copyright UCAR. This open source software is provided
+# by UCAR, "as is", without charge, subject to all terms of use at
+# http://www.image.ucar.edu/DAReS/DART/DART_download
+#
+# DART $Id$
```

Comment:
this is from the svn days. No need to add this to new files. 
```suggestion
```

Reply:
Whoops, I copied the standard mkmf.template for intel to make this. I removed the svn lines at the bottom but missed this one. Need to remove this from gfortran template too for quikscat